crates/bevy_pbr/src/render/mesh_types.wgsl

```diff
@@ -13,6 +13,12 @@ struct SkinnedMesh {
 };
 #endif
 
+#ifdef MORPH_TARGETS
+struct MorphWeights {
+    weights: array<f32, 256u>,
```

Comment:
As noted on Discord, on WebGL2, I think array members have to be 16-byte aligned. So you can either waste 12 bytes per entry, or pack them into `array<vec4<f32>, N>`.